chromecastize.sh

```diff
@@ -315,8 +329,8 @@ if ! [ -w "$CONFIG_DIRECTORY" ]; then
 fi
 
 # Load default configuration if it exists.
-if [ -f "$CONFIG_DIRECTORY/config.sh" ]; then
-  . "$CONFIG_DIRECTORY/config.sh"
+if [ -f "$CONFIG_DIRECTORY/config" ]; then
+  . "$CONFIG_DIRECTORY/config"
```

Comment:
Let's not change the name of the config file, this causes a backwards compatibility break. The config of existing users will not be loaded any more.

Reply:
Sure, makes sense. I will revert that change.